lib/search-field/index.tsx

```diff
@@ -73,21 +73,24 @@ export class SearchField extends Component<Props> {
   render() {
     const { openedTag, searchQuery } = this.props;
     const hasQuery = searchQuery.length > 0;
-    const placeholder = openedTag ?? 'Search notes and tags';
 
-    const screenReaderLabel =
-      'Search ' + (openedTag ? 'notes with tag ' : '') + placeholder;
+    const description =
+      'Search ' +
+      (openedTag ? 'notes with tag ' + openedTag : 'notes and tags');
```

Comment:
I wanted to draw attention to this change. Previously the placeholder value and the ARIA label were different strings. From reviewing the history of the UI, this at one point made sense because it appears the search field was utilized as the "title" of the list of notes, but also served a search input.

However, several changes were made to this portion of the app — the UI was moved around, the structure of the state, etc. These various changes led to an ARIA label that read "Search Search notes and tags Search notes and tags, search text field." 😂 Needless to say, this label is a bit...confusing.

So, these changes are an attempt to fix this label, but it also involves changing the _visual_ placeholder for the search input. I would appreciate if others would please look carefully at this piece to ensure the current state matches the desired experience.